### PR TITLE
fix(comment-list.js | contactAdmin): fixed es-lint errors

### PR DESCRIPTION
**Pull request by Stanislav-Vyhnan**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] All

### Second Level Review

- [ ] @VadymKharyton | @andrii-kuzmichov 

## Summary of issue

Plenty of errors by es-lint, which need to fix

## Summary of change

comment-list.js | contactAdmin: in these file and folder were fixed es-lint errors

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/containers/contactAdmin/contactAdmin-filter-container.js [hunk 1]
```diff
@@ -35,34 +41,44 @@ class ContactAdminFilterWrapper extends Component {
...
-          form_values={this.props.form_values}
-          initialFormValues={initialFormValues}
-        />
-      </>
-    );
-  }
-}
+  const initialFormValues = buildInitialFormValues();
+  return (
+    <>
```
Comment (Vladosik12333) [inline src/containers/contactAdmin/contactAdmin-filter-container.js:54]: Do you need <>?

File: src/containers/contactAdmin/contactAdmin-list-container.js [hunk 2]
```diff
@@ -64,6 +65,18 @@ class ContactAdminListWrapper extends Component {
   }
 }
 
+ContactAdminListWrapper.defaultProps = {
+  history: {},
+  getIssues: () => {},
+  contactAdminList: {},
```
Comment (Vladosik12333) [inline src/containers/contactAdmin/contactAdmin-list-container.js:71]: Maybe list is array?

**Review by Vladosik12333 — CHANGES_REQUESTED** [review]

File: src/containers/contactAdmin/contactAdmin-list-container.js (hunk 2 above)
Comment (Stanislav-Vyhnan) [reply src/containers/contactAdmin/contactAdmin-list-container.js:71]: Nope. Look on 47 line, we did destructuring object, so it's object

File: src/containers/contactAdmin/contactAdmin-filter-container.js (hunk 1 above)
Comment (Stanislav-Vyhnan) [reply src/containers/contactAdmin/contactAdmin-filter-container.js:54]: No, we don't need it, but I fixed es-lint errors only and I didn't check this things. I think, we will delete it when we do refactor this files.

Comment (Vladosik12333) [reply src/containers/contactAdmin/contactAdmin-filter-container.js:54]: I think, you can change it in this PR.

Comment (VadymKharyton) [reply src/containers/contactAdmin/contactAdmin-filter-container.js:54]: Okay, but make it as TODO

**Pull request closed without merging**